### ISSUE TITLE
代码问题

### ISSUE THREAD

**Issue by zcc0817** [YimianDai/open-acm#4]
您好，我运行代码的时候发现只有training loss的结果，但IoU和nIoU都为0，请问，是我代码有的地方没有改吗？

**Comment by YimianDai** [comment]
您好，我已经把 traning logs 给上传了，您可以查看。另外，**训练好的模型参数我一开始就已经上传了**，如若不信，**您可以自己验证一下**。代码可以参考我的另一个 repo：
https://github.com/YimianDai/open-alcnet/blob/master/visualize_local_contrast_networks.py

我在我的实验中从来没有遇到过您说的问题，我也已经提供了数据集、代码、训练好的模型，我不知道我还应该再怎么证明我自己的工作没有问题。代码不是「粉」，我也不能剖腹自证清白。当然也有可能我从我的代码库里抽出这个 repo 的时候存在我没意识到的问题，如果可以的话，您能提供更多细节吗？

「IoU 和 nIoU 都为 0」对我而言是一个非常严重的指控，而我已经尽我所能提供了 数据集、代码、训练好的模型、训练日志，足够自证清白了。而且从疑罪从无的角度，也应该由指控方提出具体的证据，我再来回应。

**Comment by zcc0817** [comment]
很抱歉会给您带来这种困扰，我并没有这种意思，我的意思是我好像是哪里没有改好，或者没有成功运行您的程序，我是想向您请教问题的，不是来指控您或者对您提出质疑的，主要我对您的代码还不熟悉，所以提出来问题请您帮我解决一下。再次跟您说声抱歉，我真的不知道会为您带来这种困扰，感谢您回复我的邮件，给您带来不便，真的很抱歉！



在 2020-12-24 22:48:05，"Yimian Dai" <notifications@github.com> 写道：

您好，我已经把 traning logs 给上传了，您可以查看。另外，训练好的模型参数我一开始就已经上传了，如若不信，您可以自己验证一下。代码可以参考我的另一个 repo：
https://github.com/YimianDai/open-alcnet/blob/master/visualize_local_contrast_networks.py

我在我的实验中从来没有遇到过您说的问题，我也已经提供了数据集、代码、训练好的模型，我不知道我还应该再怎么证明我自己的工作没有问题。代码不是「粉」，我也不能剖腹自证清白。当然也有可能我从我的代码库里抽出这个 repo 的时候存在我没意识到的问题，如果可以的话，您能提供更多细节吗？

总之，「IoU 和 nIoU 都为 0」对我而言是一个非常严重的指控，而我已经尽我所能提供了 数据集、代码、训练好的模型、训练日志，足够自证清白了。而且从疑罪从无的角度，也应该由指控方提出证据，我再来回应。

**Comment by YimianDai** [comment]
您好，我回头看一下，因为我还没收到其他人类似的反馈，这个代码放得也比较仓促，所以不太确定到底是什么原因导致您那里有这样。

等我忙完这一阵，后面会扩成 期刊版，我会用 PyTorch 把这个重新实现一遍，然后弄成一个比较像样的 benchmark dataset 和 toolkit。到时还请您多多提意见。

祝您新年快乐~

**Comment by zcc0817** [comment]
好的，非常感谢您百忙之中的回复，因为最近比较忙，所以刚看到，也祝您新年快乐！



在 2020-12-31 09:49:12，"Yimian Dai" <notifications@github.com> 写道：

您好，我回头看一下，因为我还没收到其他人类似的反馈，这个代码放得也比较仓促，所以不太确定到底是什么原因导致您那里有这样。

等我忙完这一阵，后面会扩成 期刊版，我会用 PyTorch 把这个重新实现一遍，然后弄成一个比较像样的 benchmark dataset 和 toolkit。到时还请您多多提意见。

祝您新年快乐~

**Comment by xjtuwh** [comment]
哈哈哈哈哈哈哈哈哈

**Comment by Victor4869** [comment]
> 
你是不是在windows系统下运行？ 我用windows训练就会出现 loss = NaN， 然后IoU 和 nIoU 都为 0，换成linux系统就正常了。
